modules/exploits/linux/fileformat/unrar_cve_2022_30333.rb

```diff
@@ -80,7 +80,11 @@ def exploit
     end
 
     begin
-      rar = encode_as_traversal_rar(datastore['SYMLINK_FILENAME'] || Rex::Text.rand_text_alpha_lower(4..12), datastore['TARGET_PATH'], payload_data)
+      if datastore['SYMLINK_FILENAME'] && !datastore['SYMLINK_FILENAME'].empty?
```

Comment:
Was it not possible to update this to use the same convention as the other files that you're modifying? 👀

Reply:
Got mixed up with where the other datastore option was being used - will fix 🫡